defaults/main.yml

```diff
@@ -10,44 +10,61 @@ network_provider_os_default: "{{
       'OracleLinux'
     ] and ansible_distribution_major_version is version('7', '<')
     else 'nm' }}"
+
 # If NetworkManager.service is running, assume that 'nm' is currently in-use,
 # otherwise initscripts
-network_provider_current: "{{
```

Comment:
Not sure, if this is considered an internal variable or if a user might want to be able to query this. @pcahyna what's your opinion?

Reply:
What is the case where `network_provider_current` would be different than `network_provider` or `network_provider_os_default` (https://github.com/oasis-roles/meta_standards#providers for reference)

Reply:
`network_provider_current` would be considered read-only for the user. So for example users could use an empty network configuration and then query `network_provider_current` to check what is detected. It can be differetn that the os default. For example the OS default for CentOS 7 is nm but the cloud images do not contain it as far as I know.

Not sure, if the variable is really useful for useres. I am fine with making this an internal variable but I wanted to check with you, first. If you agree with making it internal, let's do it.

Reply:
I don't have an opinion as to whether to make it internal or not - I think it depends on what you think would be useful to users of the network role.

Reply:
It does not seem to be special for the network role but for the same for any role with multiple providers. Since there is no clear request to have this variable not internal, I am in favor of making it internal.